actors/multisig/tests/util.rs

```diff
@@ -43,4 +50,53 @@ impl ActorHarness {
         rt.verify();
         ret
     }
+
+    pub fn propose_ok(
+        self: &Self,
+        rt: &mut MockRuntime,
+        to: Address,
+        value: TokenAmount,
+        method: MethodNum,
+        params: RawBytes,
+    ) -> [u8; 32] {
+        //expect_ok(self.propose(rt, to, value, method, params));
+        rt.expect_validate_caller_type(vec![*ACCOUNT_ACTOR_CODE_ID, *MULTISIG_ACTOR_CODE_ID]);
+        let propose_params =
+            ProposeParams { to: to, value: value.clone(), method: method, params: params.clone() };
```

Comment:
I think this cloning, required because the param structs move not borrow this data, is ok since the types are small but not sure if this is idiomatic.